bots/images/scripts/openshift.setup

```diff
@@ -184,7 +194,7 @@ echo '{"apiVersion":"v1","kind":"ImageStream","metadata": {"name":"busybox"},"sp
 oc create -f /tmp/imagestream.json
 
 # Get registry address and configure docker for it
-address="$(oc get services docker-registry | tail -n1 | awk -v N=2 '{print $N}')"
+address="$(oc get services docker-registry | oc get services docker-registry | grep -Eo '([0-9]{1,3}\.){3}[0-9]{1,3}')"
```

Comment:
This is redundant (although harmless), fix when rebasing after landing the two bugfix PRs.